### DISCUSSION TITLE
[Run issue]: How to using TOPAS physics module of TsPhotoNeutron?

### DISCUSSION THREAD

**Discussion by DararaiPakraypet** [OpenTOPAS/OpenTOPAS#44] [Run-Problem] [answered]
### What was the issue?

How to using TOPAS physics module of TsPhotoNeutron?

When I set the parameter of sv:Ph/Default/Modules = 1 "TsPhotoNeutron" I got the error this below:
![problem](https://github.com/OpenTOPAS/OpenTOPAS/assets/109076306/0bc8feac-a4a6-4762-a7c5-034e7783dc30)

### TOPAS category

Geometry

### Version

4.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
Have you tried using lowercase?

**Reply by DararaiPakraypet** [reply]
Thank you I can run it.
